noxfiles/ci_nox.py

```diff
@@ -0,0 +1,202 @@
+"""Contains the nox sessions used during CI checks."""
+import nox
+from constants_nox import (
+    CI_ARGS,
+    COMPOSE_FILE,
+    IMAGE_NAME,
+    INTEGRATION_COMPOSE_FILE,
+    RUN,
+    RUN_NO_DEPS,
+    START_APP,
+    WITH_TEST_CONFIG,
+)
+from docker_nox import build_local_prod
+from utils_nox import teardown
+
+RUN_STATIC_ANALYSIS = (*RUN_NO_DEPS, "nox", "-s")
+
+
+@nox.session()
+def check_all(session: nox.Session) -> None:
+    """Runs all of the CI checks."""
+    teardown(session)
+    build_local_prod(session)
+    black(session)
+    isort(session)
+    xenon(session)
+    mypy(session)
+    pylint(session)
+    check_install(session)
+    fidesctl(session)
+    fidesctl_db_scan(session)
+    pytest_unit(session)
+    pytest_integration(session)
+
+
+# Static Checks
+@nox.session()
+def black(session: nox.Session) -> None:
+    """Run the 'black' style linter."""
+    if session.posargs == ["docker"]:
+        run_command = (*RUN_STATIC_ANALYSIS, "black")
+    else:
+        run_command = ("black", "--check", "src", "tests", "noxfiles")
+    session.run(*run_command, external=True)
+
+
+@nox.session()
+def isort(session: nox.Session) -> None:
+    """Run the 'isort' import linter."""
+    if session.posargs == ["docker"]:
+        run_command = (*RUN_STATIC_ANALYSIS, "isort")
+    else:
+        run_command = ("isort", "--check-only", "src", "tests", "noxfiles")
```

Comment:
Same as with the comment on black. It would be nice to have an option to run the fix here in addition to the `--check-only`.